.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Close Stale Pull Requests
+
+on:
+  schedule:
+    #Runs at 00:00 every day
+  - cron: "0 0 * * *"
```

Comment:
With reference from [GitHub's Workflow syntax]:(https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule)
Let's standardize usage of `"` and `'` to `'`. Add a space after `# `. Also, indent two spaces for `- cron`.
```
schedule:
  # Runs at 00:00 every day
  - cron: '0 0 * * *'
```